modules/standard/Spawn.chpl

```diff
@@ -45,41 +108,74 @@ module Spawn {
   private extern proc qio_spawn_free_str(str: c_string);
 
 
-  /* A subprocess */
+  /* 
+     This record represents a subprocess.
+   */
   record subprocess {
+    /* The kind of a subprocess is used to create the types
+       for any channels that are necessary. */
     param kind:iokind;
+    /* As with kind, this value is used to create the types
+       for any channels that are necessary. */
     param locking:bool;
 
+    pragma "no doc"
     var home:locale;
 
+    /* The Process ID number of the spawned process */
```

Comment:
Process ID, and the file descriptors seems a little too much of a C concept for me.  Should we be exposing them?